plugins/txt/plugin/commands/txt_reply_cmd.rb

```diff
@@ -0,0 +1,38 @@
+module AresMUSH
+  module Txt 
+      class TxtReplyCmd
+        include CommandHandler
+
+        attr_accessor :names, :names_raw, :message, :scene_id
+
+    def parse_args
+      if (!cmd.args)
+        self.message = nil
+      else
+        self.message = cmd.args
+      end
+    end
+
+    def check_received_txts
+      unless enactor.txt_received
```

Comment:
This is only checking txt_received, but down below it looks for either txt_received or txt_received_scene.  Not sure which is correct, but I'm guessing this should probably be checking both.  Or just move this check down into handle as a final 'else'.

Reply:
The second enactor.txt_received was superfluous - I'm guessing because I think the earlier error check was added later. The checks are separate because you can text both with a scene (to a log) and without a scene. I think this is how it should be now.